dreem/training/configs/base.yaml

```
@@ -16,10 +16,11 @@ model:
   dropout_attn_head: 0.1
   embedding_meta: 
     pos:
-        mode: "fixed"
+        mode: "fixed" # supports fixed, learned, rope
```

yamllint

[warning] wrong indentation: expected 6 but found 8

(indentation)